src/backend/joanie/urls.py

```diff
@@ -247,37 +246,32 @@
         + static(settings.MEDIA_URL, document_root=settings.MEDIA_ROOT)
     )
 
+if settings.USE_SWAGGER or settings.DEBUG:
     try:
-        # Try to import `drf_yasg` dynamically as this dependency is installed only
+        # Try to import `drf_spectacular` dynamically as this dependency is installed only
```

Comment:
This is not really accurate. `drf_spectacular` is added in the `Base` settings so I think you can load it every time. So should it be added in the `Base` or not ?

Reply:
I forgot to remove those dynamic imports.